src/splitInteger.test.js

```diff
@@ -4,18 +4,19 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(6, 2))
+    .toEqual([3, 3]);
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
-
+  expect(splitInteger(6, 2)).toEqual([3, 3]);
 });
```

Comment:
Read the description of the test and check your expected result, please. 
Change it, please